bin/development-server.js

```diff
@@ -10,19 +10,13 @@ const express = require("express");
 const webpackDevMiddleware = require("webpack-dev-middleware");
 const webpackHotMiddleware = require("webpack-hot-middleware");
 const http = require("http");
+const _ = require("lodash");
 
-// Improve the first run experience: copy the local sample config to a
-// local file so webpack can find it. We don't have to do this, but
-// otherwise webpack prints a warning that scares people.
-const localConfigPath = path.join(__dirname, "../config/local.json")
-if(!fs.existsSync(localConfigPath)) {
-  const defaultConfig = fs.readFileSync(
-    path.join(__dirname, "../config/local.sample.json")
-  );
-  fs.writeFileSync(localConfigPath, defaultConfig);
-}
-
-const features = require("../config/feature");
+// Setup Config
+const getConfig = require("../config/config").getConfig;
+const features = require("../config/feature")
+const config = getConfig();
+features.setConfig(config);
```

Comment:
Sets the config in `feature` so that webpack and other server-side code has access to features. It's a small temporary hack